### PR TITLE
picolisp: fix for Xcode 9

### PR DESCRIPTION
**Pull request by fxcoudert**

### CODE REVIEW

File: Formula/picolisp.rb [hunk 1]
```diff
@@ -12,7 +12,10 @@ class Picolisp < Formula
     sha256 "7b3121d448479a6d6f1d150da4cbe0c6558626edcb7a4cbdb9d6f02230d5bf9b" => :mavericks
   end
 
+  depends_on :java => :build
+
   def install
+    ENV.delete("SDKROOT") if DevelopmentTools.clang_build_version >= 900
```
Comment (dunn) [inline Formula/picolisp.rb]: Is this related to an upstream bug? If so, could you add a comment above this line with a link to the ticket?

Comment (ilovezfs) [reply Formula/picolisp.rb]: As fx mention on Slack, this is an Apple bug supposedly fixed in the latest Xcode beta.

Comment (fxcoudert) [reply Formula/picolisp.rb]: But that's not actually the problem with picolisp :(
